intro-HPC/ch_troubleshooting.tex

```diff
@@ -108,6 +108,62 @@ \section{Issues connecting to login node}
 and include the output of that command in the message.
 \fi
 
+\ifmacORlinux
+
+\section{WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!}
+
+If you get a warning that looks like the one below, it may be possible
+that someone is trying to intercept the connection between you and the \hpc.
```

Comment:
Also mention the other (more likely?) possibility that the host keys of the system you're connecting to have been changed (for example because of system updates).